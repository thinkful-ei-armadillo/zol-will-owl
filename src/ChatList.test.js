import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Chatlist from './ChatList';
import chatEvents from './chatEvents';


describe('Messages component', () => {
  it('renders chatlist without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Chatlist chatEvents={chatEvents}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders chatlist', () => {
    const tree = renderer
      .create(<Chatlist chatEvents={chatEvents}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});