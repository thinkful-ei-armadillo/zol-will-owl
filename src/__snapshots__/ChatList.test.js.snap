// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages component renders chatlist 1`] = `
<div
  className="chatBox"
>
  <div
    className="chats"
  >
    <div
      className="chatEvent"
    >
      <div
        className="message"
      >
        <img
          alt="Axe Kubicka"
          className="avatar"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
        <span>
          Axe Kubicka
        </span>
        <span>
          12:50 PM
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Caty Flucker
        </span>
        <span>
          gave a thumbs up
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Frank Runciman
        </span>
        <span>
          gave a thumbs down
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Koren Templeton
        </span>
        <span>
          raised hand
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Koren Templeton
        </span>
        <span>
          clapped
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Koren Templeton
        </span>
        <span>
          joined
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Ashla Attwool
        </span>
        <span>
          left
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Axe Kubicka
        </span>
        <span>
          joined stage
        </span>
      </div>
    </div>
    <div
      className="chatEvent"
    >
      <div>
        <span>
          Axe Kubicka
        </span>
        <span>
          left stage
        </span>
      </div>
    </div>
  </div>
  <form
    id="chatForm"
  >
    <input
      placeholder="Chat"
      type="text"
    />
  </form>
</div>
`;
